scalafmt-benchmarks/src/resources/scala-js/Division.scala

```diff
@@ -824,7 +824,7 @@ private[math] object Division {
   /** The Montgomery modular exponentiation.
    *
    *  Implements the Montgomery modular exponentiation based in <i>The sliding
-   *  windows algorithm and the MongomeryReduction</i>.
+   *  windows algorithm and the MontgomeryReduction</i>.
```

Comment:
This is vaguely important, since there's nothing by this name. Although it's spelled properly the on the preceding lines.